.travis.yml

```diff
@@ -21,6 +21,9 @@ cache:
    - $HOME/.cache/go-build
    - $HOME/gopath/pkg/mod
 
+git:
+ depth: 10
```

Comment:
Do we need it to be this deep? Isn't 1 enough?

Reply:
ok